aws/table_aws_waf_rate_based_rule.go

```diff
@@ -0,0 +1,224 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/service/waf"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+func tableAwsWafRateBasedRule(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_waf_rate_based_rule",
+		Description: "AWS WAF Rate Based Rule",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("rule_id"),
+			ShouldIgnoreError: isNotFoundError([]string{"WAFNonexistentItemException", "ValidationException"}),
+			Hydrate:           getAwsWafRateBasedRule,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsWafRateBasedRules,
+		},
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name for the rule.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "rule_id",
+				Description: "The ID of the Rule.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "metric_name",
+				Description: "The name or description for the metrics for a RateBasedRule.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsWafRateBasedRule,
+			},
+			{
+				Name:        "rate_key",
+				Description: "The field that AWS WAF uses to determine if requests are likely arriving from single source and thus subject to rate monitoring.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsWafRateBasedRule,
+			},
+			{
+				Name:        "rate_limit",
+				Description: "The maximum number of requests, which have an identical value in the field specified by the RateKey, allowed in a five-minute period.",
+				Type:        proto.ColumnType_STRING,
```

Comment:
Seems like the value is of type integer?
Can we convert the value to `int`, or it stores the `string` value?